CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+# 0.10.0
+
+## Features
+
+- **M3Query**: Add multi-zonal capability to remote reads (#1687)
+- **M3Query**: Add debug param to namespace get API for better readability (#1698)
```

Comment:
This sentence reads a little weird. Maybe `to GET /services/m3db/namespace (or whatever the path is) API`

Reply:
Also is this documented anywhere?

Reply:
Yes, it is: http://m3db.github.io/m3/operational_guide/namespace_configuration/#viewing-a-namespace